### PR TITLE
new platform pages

### PR DESCRIPTION
**Pull request by UnsungHero97**
Figma designs:

Screen Shot 2020-10-19 at 1 55 13 PM

### CODE REVIEW

File: src/containers/platform/PlatformResearchPage.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+          </Typography>
+        </SectionContent>
+        <SectionContent margin="24px 0">
+          <Typography variant="h2">
+            While Fully Compliant
+          </Typography>
+          <Typography variant="body1">
+            Meet pretty much any compliance out there.
+          </Typography>
+        </SectionContent>
```
Comment (anzioka) [inline src/containers/platform/PlatformResearchPage.js:87]: same comment about list component

File: src/containers/platform/PlatformResearchPage.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+          <InfoTile>
+            <LockCircleIcon />
+            <Typography gutterBottom variant="h2">
+              Simplify data collaboration with one platform.
+            </Typography>
+            <Typography variant="body1">
+              With OpenLattice, researchers can securely and easily share and access data across organizational
+              boundaries. This means you can now smoothly collaborate where  previously difficult or impossible.
+            </Typography>
+          </InfoTile>
```
Comment (anzioka) [inline src/containers/platform/PlatformResearchPage.js:125]: same comment about list component

File: src/containers/platform/PlatformGovernmentPage.js [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+          </Typography>
+        </SectionContent>
+        <SectionContent margin="24px 0">
+          <Typography variant="h2">
+            While Fully Compliant
+          </Typography>
+          <Typography variant="body1">
+            Meet pretty much any compliance out there.
+          </Typography>
+        </SectionContent>
```
Comment (anzioka) [inline src/containers/platform/PlatformGovernmentPage.js:87]: no change requested, but I think this block could easily be implemented as a list component and avoid all the repetition.

**Review by anzioka — APPROVED** [review]

File: src/containers/platform/PlatformGovernmentPage.js (hunk 3 above)
Comment (UnsungHero97) [reply src/containers/platform/PlatformGovernmentPage.js]: Moved this into its own component.

**Review by anzioka — APPROVED** [review]

**Pull request merged**